packages/peregrine/lib/talons/CommunicationsPage/useCommunicationsPage.js

```diff
@@ -2,13 +2,14 @@ import { useCallback, useMemo } from 'react';
 import { useMutation, useQuery } from '@apollo/client';
 
 import { useUserContext } from '../../context/user';
+import DEFAULT_OPERATIONS from './communicationsPage.gql';
 
 export const useCommunicationsPage = props => {
+    const { afterSubmit, operations = DEFAULT_OPERATIONS } = props;
```

Comment:
Other considerations for this name: `gqlOperations`, `networkOperations`, `apolloOperations`

Reply:
We haven't decided how we're going to return our raw data, but I would want it to match this input; eg. `dataOperations` returns `data`. I hate naming things, but this should be a simple find/replace. I'll poll the team today during peer review 👍

Reply:
`gqlOperations` ✔️